go.mod

```diff
@@ -18,6 +19,7 @@ require (
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
 	google.golang.org/grpc v1.38.0
 	google.golang.org/protobuf v1.28.0
+	gopkg.in/inf.v0 v0.9.1
```

Comment:
I assume this package should still be indirect. Please re-run `go mod tidy` to confirm.

Reply:
done